net/model/NetworkDataSorce.m

```diff
@@ -139,12 +142,13 @@ -(void)requestSingUpWithUser:(User*)user
                  andDataSorceHandler:^(NSData *data, NSError *error) {
                      if(data.length >0 && error == nil)
                      {
-                         NSDictionary *userDic = [NSJSONSerialization JSONObjectWithData:data
+                         NSMutableDictionary *userDic = [[NSJSONSerialization JSONObjectWithData:data
```

Comment:
um.... When I get json from server (user or issue), and it hasn't image (avatar or attach), I'll get [NSNull null]. At few next lines I save info into NSUserDefaults and app crashes cos [NSNull null] is not a property list. So I check it, and modify Dictionary.